bundles/framework/publisher2/view/PanelGeneralInfo.js

```diff
@@ -58,6 +59,24 @@ Oskari.clazz.define('Oskari.mapframework.bundle.publisher2.view.PanelGeneralInfo
                     field.setLabel(data.label);
                     field.setTooltip(data.tooltip, data.helptags);
                     field.setPlaceholder(data.placeholder);
+
+                    const keyUpHandler = Oskari.util.throttle((event) => {
```

Comment:
Define the function outside the loop and rename it to something like domainValidatorKeyUpHandler as it's only used for domain.

Reply:
Fixed as proposed.